app/pages/lab/project-details.cjsx

```diff
@@ -30,6 +30,24 @@ module.exports = React.createClass
     backgroundError: null
     disciplineTagList: disciplineTagList
     otherTagList: otherTagList
+    avatar: null
+    background: null
+    error: null
+
+  componentWillMount: ->
+    avatar = @props.project.get('avatar')
+    background = @props.project.get('background')
+
+    Promise.all([avatar, background])
+      .then ([avatar, background]) => 
+        if avatar.src?
```

Comment:
It seems using `Promise.all` corrected the differing behavior between Chrome and Firefox, so this conditional I think can be removed.